README.rst

```diff
@@ -59,6 +59,10 @@ Install and manage `supervisor`_ configuration for the saemref installation.
 | Centos 7 | ``supervisorctl restart saemref``                   | ``systemctl restart supervisord``   |
 +----------+-----------------------------------------------------+-------------------------------------+
 
+Requirements
+============
+
+If using postgres as database driver, ensure to install the `postgres contrib`_ package.
```

Comment:
there's a `Requires:       postgresql94-contrib` in RPM spec file, how can this be not installed?